src/middlewared/middlewared/plugins/catalogs_linux/sync_catalogs.py

```diff
@@ -20,31 +20,44 @@ async def sync_all(self, job):
         catalog_len = len(catalogs)
         for index, catalog in enumerate(catalogs):
             job.set_progress((index / catalog_len) * 100, f'Syncing {catalog["id"]} catalog')
-            try:
-                await self.middleware.call('catalog.sync', catalog['id'])
-            except Exception as e:
-                self.logger.error('Failed to sync %r catalog: %s', catalog['id'], e)
+            sync_job = await self.middleware.call('catalog.sync', catalog['id'])
```

Comment:
Before this change you were logging an error, if one was raised. Now you're not. Is that intended?

Reply:
Yes that's intended